### ISSUE TITLE
One of the most recent code commits (October 9th 2015) has made the normalize GUI unusable

### ISSUE THREAD

**Issue by Mulchman** [dasilva333/TowerGhostForDestiny#425]
![example](http://i.imgur.com/kUZFvE7.png)

**Comment by Mulchman** [comment]
![example2](http://i.imgur.com/YCuWOdw.png)

**Comment by Mulchman** [comment]
Both normalize all and regular normalize are affected. I think it was one of today's commits; I haven't tracked it down to a specific commit.

**Comment by tmalfor** [comment]
When I attempt to normalize any general items the application will report back to me what it will attempt to do but nothing happens and I get the following message: "All items normalized as best as possible"

**Comment by dasilva333** [comment]
@Mulchman can you confirm the problem is still apparent in 3.3.9 or in the master's 3.4.0? Thanks

**Comment by Mulchman** [comment]
Updated to https://github.com/dasilva333/TowerGhostForDestiny/commit/d463fd158a134618f8f497b7f36b06bba65cc7be and both Normalize UIs work correctly. Normalize single and normalize all functionality appear to work correctly as well.

Also, I recommend adding a change I've been using in my branch, to avoid TTK key/rune normalizing (https://github.com/Mulchman/TowerGhostForDestiny/commit/985b45d465668ec0a22053042fd1ecbf53cd5d1c):

* **www/js/app.js** around line *1642*
``` javascript
                  while (description !== undefined) {
                     if ((description !== "Hadronic Essence") &&
                         (description !== "Sapphire Wire") &&
-                        (description !== "Plasteel Plating")) {
+                        (description !== "Plasteel Plating") &&
+                        (description.indexOf(" Rune") < 0) &&
+                        (description.indexOf("Key of") < 0)) {
                         break;
                     } else {
+                        //tgd.localLog("normalizeAll: skipping '" + description + "'");
                         description = getNextDescription();
                     }
                 }
```

**Comment by dasilva333** [comment]
Expect the problem fixed in 3.4.0 I just rewrote that whole section it was garbage anyways, expect it to run better now. Thanks
